parsing/parser.mly

```diff
@@ -1304,7 +1303,7 @@ class_type_parameters:
   | LBRACKET type_parameter_list RBRACKET       { $2 }
 ;
 
-class_fun_def: mkclass(class_fun_def_desc) { $1 };
+%inline class_fun_def: mkclass(class_fun_def_desc) { $1 };
```

Comment:
I would inline `class_fun_def_desc` instead, or even use the weird-looking `) { $1 }` pattern.